Arrays and Cycles/Arrays and Cycles.js

```diff
@@ -39,51 +38,50 @@ console.log(arr);
 // 5.6 Отфильтровать массив [1,6,7,8,3,4,5,6] по значению >3
 
 function isBig(numb) {
-  return numb > 3;
+    return numb > 3;
 }
 
 let filtered = [1, 6, 7, 8, 3, 4, 5, 6].filter(isBig);
 console.log(filtered);
 // 5.7 Написать функцию, которая принимает два параметра - массив и число и выводит индекс элемента массива равный числу
 
 function getElementIndex(array, element) {
-  return array.indexOf(element);
+    return array.indexOf(element);
 }
 // 5.8 Реализовать цикл, который будет выводить число а, пока оно не станет меньше 10
 
 function C(a) {
```

Comment:
the function should have a clear name, С is a very bad name for a function. By naming convention, a function must begin with a verb because it is an action.